easybuild/easyblocks/generic/bundle.py

```diff
@@ -321,8 +320,37 @@ def install_step(self):
                             new_val = path
                         env.setvar(envvar, new_val)
 
-            # close log for this component
-            comp.close_log()
```

Comment:
We should make sure that this removal is safe. Based on the changes in framework I think it is, as the components just use the log file of the Bundle. Keeping it in doesn't hurt though, as the changes in framework should make sure that only Bundle can close its log and none of the components.

Reply:
I don't see the point in keeping this, it's clear that the component shouldn't be closing the log now since it's using the "parent" log provided by the `Bundle` instance.